### DISCUSSION TITLE
Stable Cascade

### DISCUSSION THREAD

**Discussion by marcfpf** [lllyasviel/Fooocus#2249] [Ideas]
Key Takeaways:

```
Today we are releasing Stable Cascade in research preview, a new text to image model building upon the [Würstchen](https://openreview.net/forum?id=gU58d5QeGv) architecture. This model is being released under a non-commercial license that permits non-commercial use only.

Stable Cascade is exceptionally easy to train and finetune on consumer hardware thanks to its three-stage approach.

In addition to providing checkpoints and inference scripts, we are releasing scripts for finetuning, ControlNet, and LoRA training to enable users further to experiment with this new architecture that can be found on [the Stability GitHub page.](https://github.com/Stability-AI/StableCascade)
```
https://stability.ai/news/introducing-stable-cascade

**Comment by mashb1t** [comment]
> The model is available for inference in the [diffusers library](https://huggingface.co/stabilityai/stable-cascade).

Fooocus does not use diffusers as a backend, so this would need further development and adjustments of pipelines.

**Reply by yhyu13** [reply]
sad

**Reply by DavidAllenNeron** [reply]
but it's totally doable? 

It would be the best.

**Comment by barepixels** [comment]
**Native stable cascade support for comfyui just dropped**
https://www.reddit.com/r/comfyui/comments/1aswyul/native_stable_cascade_support_for_comfyui_just/

I did a 1920x1152 on 3090 Prompt executed in 21.56 seconds
![native-stable-cascade-support-for-comfyui-just-dropped-v0-cnaupwsbj6jc1](https://github.com/lllyasviel/Fooocus/assets/92839033/48913b26-fee5-4f21-b5bf-b70e103ed701)

Workflow https://flowt.ai/community/stable-cascade-basic-workflow-pjk1r-v
files are safetensors

**Comment by DavidAllenNeron** [comment]
OMG please please please allow Fooocus AI to use CASCADE :P 

It'll be so amazing.

**Comment by jaimeastorga2000** [comment]
Unlike SDXL, Stable Cascade is _not_ open source/free software; it has a non-commercial license, much like SDXL Turbo and SVD.

**Reply by MetatronR** [reply]
https://stability.ai/membership
Both SDXL Turbo and Stable Video Diffusion can be used commercially. (https://stability.ai/core-models)

Yes, they require a membership in order to be legally used for commercial purposes, but it's a false claim to say that they have a non-commercial license.

Also, [emad_9608 (CEO)](https://www.reddit.com/user/emad_9608/) said that all experimental architectures will have a  [non-commercial license first](https://www.reddit.com/r/StableDiffusion/comments/1aqdd6j/comment/kqd43mz/).

> New models architectures (for us) are NC then we change usually, which will be in this case.
This is like a research preview basically.

Right now, the Stable cascade architecture requires 20+ GBs of RAM, yet they expect to make it work better in the [near future](https://www.reddit.com/r/StableDiffusion/comments/1aprm4j/comment/kq9xsy5/).

> original SD used way more, I would imagine this would be < 8gb VRAM in a week or two

While this comment sounds a bit too optimistic too me, what I get from him is that the 'final' product is not out yet, that all new models will be included in their membership system and that we should wait for a more optimized version.

He also said that he intends to make the licensing/membership conditions more clear [in the future](https://www.reddit.com/r/StableDiffusion/comments/1aqdd6j/comment/kqcx74v/).

> Q:
Will you stay in the confusing "commercial use" nomenclature?
I work for a nonprofit organisation, and we have funding, not revenue. It's a mess each time some license language is about commercial use (including the infamous Creative Commons- Noncommercial), cause there is no consensus on whether it's legally a commercial use to accept donations.
So... Are organisations like Greenpeace, WHO, NASA and that dude that opened a Patreon for a hobby project considered "commercial use" in your mind?

> A:
Yeah will update memberships soon

**Also - personal opinion**:
I can't understand how people expect a ML/AI company not to get bankrupt without charging for its products.
The fact that the membership for a **<$1 million** legal entity is just $20 is pretty generous.
